internal/provider/nomad/task.go

```diff
@@ -112,6 +115,7 @@ func (c *Client) listTaskImages() []model.Image {
 							Str("task_group", *taskGroup.Name).
 							Str("task_name", task.Name).
 							Str("image_name", imageName).
+							Str("tags", fmt.Sprintf("%v", labels)).
```

Comment:
```suggestion
							Strs("tags", labels).
```

Reply:
I could create a sub dict and loop through pairs and add them as Strs.

Reply:
> I could create a sub dict and loop through pairs and add them as Strs.

Yes sgtm, thx

Reply:
Or, this may be too noisy and I can remove it again. I added it so I could debug this issue and figured it may be a good tool for sanity checking, bit it could also be pretty noisy.

Reply:
Yes I was thinking the same actually after looking at the expected output. Let's just remove it from the logs then :+1:

Reply:
So I just looked at the Swarm provider and it does add the labels there too and uses `Interface()`. That seems to work here as well.

Reply:
Ah yeah right forgot about, lgtm like this.